test/fake_reply.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+module.exports = function () {
```

Comment:
unclear what this `fake_reply` is _for_ ... a `JSDoc` would be _really helpful_ here...